src/client/Microsoft.Identity.Client/AppConfig/TraceTelemetryConfig.cs

```diff
@@ -23,7 +25,7 @@ public class TraceTelemetryConfig : ITelemetryConfig
         /// <remarks>This API is experimental and it may change in future versions of the library without an major version increment</remarks>
         public TraceTelemetryConfig()
         {
-            SessionId = Guid.NewGuid().AsMatsCorrelationId();
+            SessionId = Guid.NewGuid().ToString();
```

Comment:
>SessionId [](http://example.com/codeflow?start=12&length=9)

if the class is never used perhaps this should be a fixed string. such as string.empty or 0 mor Guid.empty?

Reply:
Would it even matter if the class is never instantiated/?